management-api/base/deploymentconfig.yaml

```diff
@@ -88,11 +88,6 @@ spec:
                 configMapKeyRef:
                   key: bucket-prefix
                   name: ceph
-            - name: THOTH_SECRET
```

Comment:
i didnt see this being used, so removed it.

Reply:
Verified, it is not used anywhere in sources 👍🏻